service/worker/deployment/deployment_activities.go

```diff
@@ -67,21 +60,39 @@ func (a *DeploymentActivities) StartDeploymentSeriesWorkflow(ctx context.Context
 		Memo: map[string]interface{}{
 			DeploymentSeriesBuildIDMemoField: "",
 		},
-		WorkflowIDReusePolicy:    enums.WORKFLOW_ID_REUSE_POLICY_ALLOW_DUPLICATE,
-		WorkflowIDConflictPolicy: enums.WORKFLOW_ID_CONFLICT_POLICY_USE_EXISTING,
+		WorkflowIDReusePolicy:    enumspb.WORKFLOW_ID_REUSE_POLICY_ALLOW_DUPLICATE,
+		WorkflowIDConflictPolicy: enumspb.WORKFLOW_ID_CONFLICT_POLICY_USE_EXISTING,
 	}
 
-	// Build workflow args
-	deploymentSeriesWorkflowArgs := &deployspb.DeploymentSeriesWorkflowArgs{
+	// Calling the workflow with the args
+	_, err := a.sdkClient.ExecuteWorkflow(ctx, workflowOptions, DeploymentSeriesWorkflow, &deploymentspb.DeploymentSeriesWorkflowArgs{
 		NamespaceName: a.namespaceName.String(),
 		NamespaceId:   a.namespaceID.String(),
+	})
+	if err != nil {
+		logger.Error("starting deployment series workflow failed", "seriesName", input.SeriesName, "error", err)
 	}
+	return err
+}
 
-	// Calling the workflow with the args
-	_, err := sdkClient.ExecuteWorkflow(ctx, workflowOptions, DeploymentSeriesWorkflow, deploymentSeriesWorkflowArgs)
+func (a *DeploymentActivities) SyncUserData(ctx context.Context, input *deploymentspb.SyncUserDataRequest) error {
+	logger := activity.GetLogger(ctx)
+	logger.Info("syncing task queue userdata for deployment", "taskqueue", input.TaskQueueName, "type", input.TaskQueueType,
+		"seriesname", input.Deployment.SeriesName, "buildid", input.Deployment.BuildId)
+
+	_, err := a.matchingClient.SyncDeploymentUserData(ctx, &matchingservice.SyncDeploymentUserDataRequest{
+		NamespaceId:   a.namespaceID.String(),
+		TaskQueue:     input.TaskQueueName,
+		TaskQueueType: input.TaskQueueType,
+		Deployment:    input.Deployment,
+		Data:          input.Data,
+	})
 	if err != nil {
-		return err
+		logger.Error("syncing task queue userdata", "error", err)
```

Comment:
should we add the deployment and tq info to the log?

Reply:
yeah, though it's possible to correlate with the Info log above by activity id (which is already in the logger).

actually, on second thought: the sdk workflow/activity logs are already tagged with workflow id, which has the deployment series+buildid at a glance. so I'll remove those from the explicit logs.